docs/source/conf.py

```diff
@@ -12,21 +12,20 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import os
-import sys
-#sys.path.insert(0, os.path.abspath('../src'))
+
+# sys.path.insert(0, os.path.abspath('../src'))
 
 
 # -- Project information -----------------------------------------------------
 
-project = 'caustics'
-copyright = '2023, Ciela Institute'
-author = 'Ciela Institute'
+project = "caustics"
+copyright = "2023, Ciela Institute"
+author = "Ciela Institute"
 
 # The short X.Y version
-version = '0.5'
+version = "0.5"
```

Comment:
I'm not sure how the version number updates now, but I imagine this update means we should increment the version number to 0.6.0, right?

Reply:
I'm simply just porting this over from the current setup. A separate PR for docs stuff is coming 🙃